sites/svelte.dev/svelte.config.js

```diff
@@ -8,6 +10,23 @@ export default {
 		prerender: {
 			// TODO: REMOVE
 			handleMissingId: 'ignore',
+			entries: ['*', ...replJsonEntries(), ...tutorialEntries()],
 		},
 	},
 };
+
+/** @returns {('*' | `/${string}`)[]} */
+function replJsonEntries() {
+	// @ts-ignore
+	return get_examples_list(
+		get_examples_data(new URL('../../site/content/examples', import.meta.url).pathname)
+	).flatMap(({ examples }) => examples.map(({ slug }) => `/repl/${slug}.json`));
+}
+
+/** @returns {('*' | `/${string}`)[]} */
+function tutorialEntries() {
+	// @ts-ignore
+	return get_tutorial_list(
+		get_tutorial_data(new URL('../../site/content/tutorial', import.meta.url).pathname)
+	).flatMap(({ tutorials }) => tutorials.map(({ slug }) => `/tutorial/${slug}`));
+}
```

Comment:
note that elliott is working on a more idiomatic approach to this which we'll be able to use soon instead of this https://github.com/sveltejs/kit/pull/9571

Reply:
That would make things so much easier 🤩! The second that feature is released I'll move the code to that

Reply:
@Rich-Harris you're right, the approach used in [sveltejs/kit#9571](https://github.com/sveltejs/kit/pull/9571) is much easier to understand.